.github/workflows/locize-sync.yml

```diff
@@ -0,0 +1,56 @@
+name: Translation sync added and deleted keys to locize
+on: workflow_dispatch
+
+jobs:
+  locize-sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: 'https://registry.npmjs.org'
+          always-auth: true
+
+      - name: Setup Cognite CICD NPM
+        run: |
+          echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> .npmrc
+          npm whoami
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
```

Comment:
Don't think this is nessesary since we are authenticating with yarn, and are only using yarn afaik.

Reply:
`yarn` would install `@cognite/cogs.js` which would need authenticating